types/stream-counter/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for stream-counter 1.0
+// Project: https://github.com/andrewrk/node-stream-counter
+// Definitions by: Matthew Peveler <https://github.com/MasterOdin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Writable } from "stream";
+
+declare class StreamCounter extends Writable {
```

Comment:
The symbol is called 'ByteCounter', please do not change symbol names:
```js
Counter = require('stream-counter')
[Function: ByteCounter]
> c = new Counter()
ByteCounter {
....
```
(REPL)